### PR TITLE
Modify binary jar for prebuilt aars to come from unzip aar directory

### PR DESCRIPTION
**Pull request by kageiit**
When running buck project, `res` folder is available from the unzipped aar directory already. This modifies the path to `classes.jar` to also come from the same location and avoid having to build a separate flavor of android_prebuilt_aar

This change is required to support Android Layout Preview in buck project and intellij

### CODE REVIEW

**Comment by kageiit** [comment]
This change is required to support Android Layout Preview in buck

**Comment by kageiit** [comment]
@christolliday any update on this?

File: src/com/facebook/buck/features/project/intellij/DefaultIjLibraryFactory.java [hunk 1]
```diff
@@ -172,6 +168,7 @@ public void apply(
       // Based on https://developer.android.com/studio/projects/android-library.html#aar-contents,
       // the AAR library is required to have a resources folder.
       library.addClassPaths(Paths.get(aarUnpackPath.toString(), "res"));
+      library.addBinaryJars(Paths.get(aarUnpackPath.toString(), "classes.jar"));
```
Comment (artem-zinnatullin) [inline src/com/facebook/buck/features/project/intellij/DefaultIjLibraryFactory.java]: yis yis, nice

**Comment by christolliday** [comment]
Nice! please update the two integration tests and I'll merge

**Comment by kageiit** [comment]
> Nice! please update the two integration tests and I'll merge

Hey @christolliday , There are no integration tests for aars anywhere (search yielded 0 hits on "aar" inside the test imls). Can you point me to which tests you are talking about?

**Comment by kageiit** [comment]
I ran `ProjectIntegrationTest` on this change with no failures

**Comment by kageiit** [comment]
@christolliday Fixed the tests

**Pull request closed without merging**